### ISSUE TITLE
[Feature Request] Being able to set that additional software gets updated automatically

### ISSUE THREAD

**Issue by Toadfield** [PhoenicisOrg/phoenicis#2363]
For example,if you want that every dxvk or whatever gets updated automatically for the prefixes.
That would be useful cause I hate to update the dxvk version manually for all prefixes (how can I do that at all in phoenicis?).
And if I don't update it,then I don't have the best possible performance.

**Comment by plata** [comment]
The problem is: Phoenicis cannot even know which additional software is installed in a certain prefix. So how should it update it?

**Comment by Toadfield** [comment]
Is there a way to add that phoenicis can know which additional software is installed there somehow?

**Comment by plata** [comment]
Not really (at least not reliably).

**Comment by Toadfield** [comment]
I mean can't it scan the prefixes the whole time and if it doesn't find the files that the additional software creates?

**Comment by plata** [comment]
No. This would require to know exactly which files are created by any possible software (including all versions). That's just unrealistic.
